packages/eslint-config-react/rules/react-intl.js

```diff
@@ -72,13 +72,6 @@ module.exports = {
     */
     'formatjs/no-offset': ['error'],
 
-    /*
```

Comment:
why did you remove this?

Reply:
<img width="671" alt="image" src="https://user-images.githubusercontent.com/363385/93753776-fa6f0200-fc00-11ea-8482-594db0859b04.png">

It is not supported by upstream anymore

Reply:
oh, that was a short-lived rule then :)